### ISSUE TITLE
Game Crash when enemy has no equipment

### ISSUE THREAD

**Issue by kittlish** [TerryCavanagh/diceydungeons.com#1414]
I'm playing parallel universe Inventor, and I got the Reversal gadget. So, naturally, I unequipped everything, and went into the next battle (a super wizard). I swapped equipment, and was having a grand old time beating up this helpless old man when suddenly the game crashed. Here's the crash report:
```
--------------------------------------
filename:	diceydungeons
package:	com.terrycavanaghgames.diceydungeons
version:	v1.3.2 / 0.11.0 (mod API)
sess. ID:	dicey_dungeons_2019-09-07_14'29'19
started:	2019-09-07 14:29:19
--------------------------------------
crashed:	2019-09-07 17:17:07
duration:	02:47:47
error:		ERROR in callscenemethod(Combat,update) static : Null Object Reference, stack = 
Called from elements.Fighter.applyequipmentcurses (elements/Fighter.hx line 865)
Called from elements.CombatCommand.execute (elements/CombatCommand.hx line 128)
Called from states.Combat.update (states/Combat.hx line 268)
Called from Reflect.callMethod (C:\coding\haxe\haxe\std/cpp/_std/Reflect.hx line 55)
stack:
haxegon.Scene.callscenemethod (haxegon/Scene.hx line 93)
haxegon.Scene.update (haxegon/Scene.hx line 53)
haxegon.Core.doupdate (haxegon/Core.hx line 268)
haxegon.Core.onEnterFrame (haxegon/Core.hx line 182)
starling.events.EventDispatcher.__invokeEvent (starling/events/EventDispatcher.hx line 184)
starling.events.EventDispatcher.dispatchEvent (starling/events/EventDispatcher.hx line 144)
starling.display.DisplayObject.dispatchEvent (starling/display/DisplayObject.hx line 778)
starling.display.DisplayObjectContainer.broadcastEvent (starling/display/DisplayObjectContainer.hx line 449)
starling.display.Stage.advanceTime (starling/display/Stage.hx line 108)
starling.core.Starling.advanceTime (starling/core/Starling.hx line 469)
starling.core.Starling.nextFrame (starling/core/Starling.hx line 455)
starling.core.Starling.onEnterFrame (starling/core/Starling.hx line 682)
openfl.events.EventDispatcher.__dispatchEvent (openfl/events/EventDispatcher.hx line 443)
openfl.display.DisplayObject.__dispatch (openfl/display/DisplayObject.hx line 1236)
```

From further testing, the crash only happens when attempting to poison him. Nothing wrong with shocking him, but the parallel universe poison doesn't know what to do without any equipment to target.

Also, when the game crashed, I got the dialogue telling me where to find the crash report, but clicking "OK" didn't actually quit the game; I had to open task manager to kill it. Not sure if that's universal.

**Comment by KBKarma** [comment]
This happened to me as well against a Haunted Jar. I didn't have to force close the game, but I also had the crash on poisoning. I got around it by having one thing equipped, then swapping. Here's the crash log:

```
--------------------------------------
filename:	diceydungeons
package:	com.terrycavanaghgames.diceydungeons
version:	v1.4.3 / 0.11.0 (mod API)
sess. ID:	dicey_dungeons_2019-09-21_16'48'07
started:	2019-09-21 16:48:07
--------------------------------------
crashed:	2019-09-21 17:05:55
duration:	00:17:47
error:		ERROR in callscenemethod(Combat,update) static : Null Object Reference, stack = 
Called from elements.Fighter.applyequipmentcurses (elements/Fighter.hx line 868)
Called from elements.CombatCommand.execute (elements/CombatCommand.hx line 128)
Called from states.Combat.update (states/Combat.hx line 270)
Called from Reflect.callMethod (C:\coding\haxe\haxe\std/cpp/_std/Reflect.hx line 55)
stack:
haxegon.Scene.callscenemethod (haxegon/Scene.hx line 93)
haxegon.Scene.update (haxegon/Scene.hx line 53)
haxegon.Core.doupdate (haxegon/Core.hx line 268)
haxegon.Core.onEnterFrame (haxegon/Core.hx line 182)
starling.events.EventDispatcher.__invokeEvent (starling/events/EventDispatcher.hx line 184)
starling.events.EventDispatcher.dispatchEvent (starling/events/EventDispatcher.hx line 144)
starling.display.DisplayObject.dispatchEvent (starling/display/DisplayObject.hx line 778)
starling.display.DisplayObjectContainer.broadcastEvent (starling/display/DisplayObjectContainer.hx line 449)
starling.display.Stage.advanceTime (starling/display/Stage.hx line 108)
starling.core.Starling.advanceTime (starling/core/Starling.hx line 469)
starling.core.Starling.nextFrame (starling/core/Starling.hx line 455)
starling.core.Starling.onEnterFrame (starling/core/Starling.hx line 682)
openfl.events.EventDispatcher.__dispatchEvent (openfl/events/EventDispatcher.hx line 443)
openfl.display.DisplayObject.__dispatch (openfl/display/DisplayObject.hx line 1236)
```

**Comment by TerryCavanagh** [comment]
This one should be fixed in v1.5! Thank you :)
